File: Results/DER/Translation/translation_output/no_test/llama2/codenet/java/python-sanitized/p04043.py
import java.util.Scanner

class p04043:
    def main(self):
        sc = Scanner(System.in)
        a = sc.nextInt()
        b = sc.nextInt()
        c = sc.nextInt()

        if a == 5 and b == 7 and c == 5:
            print("YES")
        elif a == 5 and b == 5 and c == 7:
            print("YES")
        elif a == 7 and b == 5 and c == 5:
            print("YES")
        else:
            print("NO")

File: Results/DER/Translation/translation_output/misleading_test/deepseekcoder/codenet/python/java-sanitized/p04043.java
import java.util.Arrays;
import java.util.Scanner;

public class p04043 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        if (isSorted(new int[]{a, b, c})) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }
    }

    private static boolean isSorted(int[] array) {
        int[] sortedArray = Arrays.copyOf(array, array.length);
        Arrays.sort(sortedArray);
        return Arrays.equals(array, sortedArray);
    }
}